### PR TITLE
BUGFIX: RAIL-934 RAIL-850 Backend info & IE11 tabs

### PR DESCRIPTION
**Pull request by czechian**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/241/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1480/ : SUCCESS in 3m 35s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/242/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1481/ : SUCCESS in 3m 30s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/272/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1511/ : FAILURE in 3m 50s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/273/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1512/ : SUCCESS in 4m 06s

File: examples/src/components/utils/Header.jsx [hunk 1]
```diff
@@ -125,25 +169,97 @@ class Header extends React.Component {
...
+                        padding: 0;
+                    }
+
+                    .page :global(.backendInfoValue) {
+                        display: inline-block;
+                        margin-left: 4px;
+                        color: #333;
+                    }
+
+                    .page :global(.backendInfoClose) {
```
Comment (tomasreichmann) [inline examples/src/components/utils/Header.jsx]: .backendInfoClose doesn't need to be :global

**Comment by tomasreichmann** [comment]
LGTM

**Comment by tomasreichmann** [comment]
@czechian, check the gate. I haven't seen this status before.

**Comment by tomasreichmann** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/358/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1593/ : SUCCESS in 4m 48s

File: examples/src/components/utils/Header.jsx (hunk 1 above)
Comment (czechian) [reply examples/src/components/utils/Header.jsx]: Really? backendinfo

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/58/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1595/ : SUCCESS in 8m 39s
- https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/646/ : SUCCESS in 14m 55s

**Pull request merged**